templates/CommentsInterface_singlecomment.ss

```diff
@@ -34,7 +34,11 @@
 				<% end_if %>
 			</div>
 			<% if $RepliesEnabled %>
-				<a class="comment-reply-link" href="#{$ReplyForm.FormName}">Reply to $AuthorName.XML</a>
+                <% if $JavaScriptEnabled %>
+				    <a class="comment-reply-link" href="#{$ReplyForm.FormName}">Reply to $AuthorName.XML</a>
+                <% else %>
+                    <a class="comment-reply-link" href="$Parent.Link?replyTo=$ID#Form_ReplyForm_$ID">Reply to $AuthorName.XML</a>
```

Comment:
Would it be classy if the javascript intercepted this link, but loaded in the form dynamically? :P It would keep the page HTM light, but still prevent having to navigate.

We actually went through this thought process quite a lot when developing the original feature; Although I must say that your replyTo form action is quite a nice solution, one that I wish we'd considered.